Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.cpp

```diff
@@ -1728,8 +1748,31 @@ bool RemCalculationNode::equals(CalculationNode const& other) const
         && m_y->equals(*static_cast<RemCalculationNode const&>(other).m_y);
 }
 
+static Optional<CSSNumericType> numeric_type_from_value(CalculatedStyleValue::CalculationResult::Value const& value)
+{
+    return value.visit(
+        [](Number const&) -> Optional<CSSNumericType> { return {}; },
```

Comment:
I don't know what `CSSNumericType` should occur in the case of `Number`, I assume there isn't a set one and it can change. But this code should probably be corrected if there is one.